pkg/crc/ssh/client.go

```diff
@@ -54,15 +54,17 @@ func clientConfig(user string, keys []string) (*ssh.ClientConfig, error) {
 		}
 
 		privateKeys = append(privateKeys, privateKey)
+		keyPaths = append(keyPaths, k)
 	}
 
-	if len(privateKeys) > 0 {
-		authMethods = append(authMethods, ssh.PublicKeys(privateKeys...))
+	if len(privateKeys) == 0 {
+		return nil, errors.New("no ssh private keys available")
 	}
+	log.Debugf("Using ssh private keys: %v", keyPaths)
```

Comment:
For what it's worth, since commit 81268bf2f2, we have a single ssh session for each crc run, so during initial start, only the key from `~/.crc/cache` will be used, and shown here. During subsequent starts, only the key from `~/.crc/machines/crc` will be usable, the other one has been removed from `authorized_keys`. This code will be printing the 2 keys, but only the one from `~.crc/machines/crc` will be usable.

Mentioning this here because it confused me and I was not sure the code was working as expected or not (I thought it would be switching to the new ssh key once it's generated).